.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 ## Why
 
-[Relevant ticket](https://broadinstitute.atlassian.net/browse/DSPDC-1352)
+[Relevant ticket](https://broadinstitute.atlassian.net/browse/<ticket_id>)
```

Comment:
Should we be creating one of these files per project for new PRs moving forward?

Reply:
I think so, yes.